Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -2650,10 +2661,21 @@ auto RenderObject::collectSelectionGeometriesInternal(const SimpleRange& range)
     bool containsDifferentWritingModes = false;
     bool hasLeftToRightText = false;
     bool hasRightToLeftText = false;
+    bool isInDifferentFlow = false;
```

Comment:
what does **_flow_** mean in this context? Different RenderBlockFlow? Different formatting context subtree? Inflow vs. out-of-flow? something else?

Reply:
It means inflow vs. out-of-flow, how about `hasFlowTransition`?

Reply:
We replaced it with `needsToSeparateBoundary` here, and reveal the position info in functions `currentNodeRequiresToSeparateBoundary` and `lastNodeRequiresToSeparateBoundary`. Hope it looks clearer now:)